pkgs/desktops/plasma-5/kwin/default.nix

```diff
@@ -40,6 +40,8 @@ mkDerivation {
     ./0002-xwayland.patch
     ./0003-plugins-qpa-allow-using-nixos-wrapper.patch
     ./0001-NixOS-Unwrap-executable-name-for-.desktop-search.patch
+    # https://github.com/NixOS/nixpkgs/issues/143272
+    ./0004-wayland-pass-some-environment-variables-through-args.patch
```

Comment:
Can you expand the comment to include a short sentence what this fixes?

Also please fetch https://invent.kde.org/plasma/kwin/-/commit/9a008b223ad696db3bf5692750f2b74e578e08b8.diff

Reply:
[The MR](https://invent.kde.org/plasma/kwin/-/merge_requests/1590) is not merged yet. If there is any more modification and force push, the URL would be broken.

Reply:
https://invent.kde.org/plasma/kwin/-/merge_requests/1590.diff would be broken
but the commit diff wont be